content/posts/2021/release-0.8.0.md

```diff
@@ -0,0 +1,363 @@
+---
+title: "REANA 0.8.0 is released"
+date: 2021-11-30T10:19:06+01:00
+tags:
+  - release
+  - client
+  - cluster
+---
+
+We are glad to announce the release of REANA 0.8.0, a major update
+allowing users to run Snakemake workflows, allowing administrators to
+set and monitor CPU and Disk quotas for users, and more!
+
+<!--more-->
+
+
+## What's new for the users?
+
+### Support for running Snakemake workflows
+
+One of the most notable new features of REANA 0.8 release series is
+the support for running [Snakemake](https://snakemake.github.io/)
+workflows. Snakemake joins CWL and Yadage as another workflow
+specification language that you can use to write complex computational
+workflows:
+
+```yaml
+# Snakefile
+rule all:
+    input:
+        "results/data.root",
+        "results/plot.png"
+
+rule gendata:
+    input:
+        gendata_tool=config["gendata"]
+    output:
+        "results/data.root"
+    params:
+        events=config["events"]
+    container:
+        "docker://reanahub/reana-env-root6:6.18.04"
+    shell:
+        "mkdir -p results && root -b -q '{input.gendata_tool}({params.events},\"{output}\")'"
+
+rule fitdata:
+    input:
+        fitdata_tool=config["fitdata"],
+        data="results/data.root"
+    output:
+        "results/plot.png"
+    container:
+        "docker://reanahub/reana-env-root6:6.18.04"
+    shell:
+        "root -b -q '{input.fitdata_tool}(\"{input.data}\",\"{output}\")'"
+```
+
+Please see the [dedicated blog
+post](/posts/2021/support-for-running-snakemake-workflows) about how
+to use Snakemake with REANA.
+
+### Upgrade of CWL workflow engine
+
+The Common Workflow Language engine was upgraded from `cwltool`
+version 1 to version 3 and the compliance with the CWL reference test
+suite has been improved. This allows using many new CWL constructs in
+your workflows.
+
+### CPU and Disk quota accounting
+
+Optionally, the REANA cluster may have CPU and Disk quota accounting
+feature turned on. The new command `reana-client quota-show` will
+allow seeing your quota limits and current usage:
+
+```console
+$ reana-client quota-show --resource cpu --report limit
+No limit.
+$ reana-client quota-show --resource disk --report limit
+No limit.
+```
```

Comment:
Here, there's no space between the two commands whereas in the next example there is, we should be consistent. No strong opinion on which one looks better.

Reply:
Added, thanks.